hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigurationAwareConfig.java

```diff
@@ -1255,4 +1256,9 @@ private ExternalDataStoreConfig getExternalDataStoreConfigInternal(String name,
     public ExternalDataStoreConfig findExternalDataStoreConfig(String name) {
         return new ExternalDataStoreConfigReadOnly(getExternalDataStoreConfigInternal(name, "default"));
     }
+
+    @Override
```

Comment:
We don't want any dynamic configuration for Alto. The whole design is a mess.

Reply:
This is not about dynamic config. Config object is wrapped with this class. So if you want your node.getConfig() to see AloConfig we need this method. Otherwise we would need to call node.getConfig().getStaticConfig() everywhere.

Reply:
Can't we throw an exception.